Cython/Compiler/Nodes.py

```diff
@@ -5502,6 +5504,7 @@ def generate_type_ready_code(entry, code, bases_tuple_cname=None, check_heap_typ
                         code.putln(preprocessor_guard)
                     code.putln('#if CYTHON_COMPILING_IN_CPYTHON')
                     code.putln("{")
+                    code.putln('/* The docstring is set after PyType_Ready, which may not work on non-cpython */')
```

Comment:
As a general rule we drop most comments from the utility code so we don't leave a lot in the generated code. There has to be a fairly good reason to want users to see it to leave a comment in the generated C (e.g. https://github.com/cython/cython/pull/3582, which is deliberately generating a compiler warning if the users do something wrong, but also leaves a comment for them to try to explain what to do).

With that in mind, it might be better as a Python comment, rather than a C comment)

Reply:
ok